index/scorch/segment/mem/segment.go

```diff
@@ -107,59 +107,90 @@ func New() *Segment {
 func (s *Segment) updateSizeInBytes() {
 	var sizeInBytes uint64
 
+	// FieldsMap, FieldsInv
 	for k, _ := range s.FieldsMap {
 		sizeInBytes += uint64(len(k)*2 /* FieldsMap + FieldsInv */ +
```

Comment:
need additional constant SizeOfString to account for string header (16 bytes on 64bit)

then we need (len(k)+SizeOfString)*2

Reply:
Done.